docs/data-sources/cts_notifications.md

```diff
@@ -0,0 +1,94 @@
+---
+subcategory: "Cloud Trace Service (CTS)"
+---
+
+# huaweicloud_cts_notifications
+
+Use this data source to get the list of CTS key event notifications.
```

Comment:
Use this data source to get the list of CTS key event notifications within HuaweiCloud.

Reply:
fixed.